website/docs/property/box_fit.md

```diff
@@ -0,0 +1,33 @@
+### BoxFit
```

Comment:
Can we rename the `property` folder to `styles`. And can you please add the `/_category_.json` file for it just like we did for `widgets`
 https://github.com/BuildMirai/mirai/blob/dev/website/docs/widgets/_category_.json